HacktoberfestProject.Web/Data/TableContext.cs

```diff
@@ -11,149 +11,149 @@
 
 namespace HacktoberfestProject.Web.Data

```

Comment:
What's the change here? Formatting?

Reply:
i'm not sure  its probably because my editor is setup to deafult to Tabs not Spces for accsesability issues. This article quite cleanly explains  why I use tabs over spaces 
 [https://dev.to/alexandersandberg/why-we-should-default-to-tabs-instead-of-spaces-for-an-accessible-first-environment-101f ](https://dev.to/alexandersandberg/why-we-should-default-to-tabs-instead-of-spaces-for-an-accessible-first-environment-101f)

Reply:
I've listened to Jeff Fritz talk about how tabs are more accessible than spaces. He also pointed to the Reddit post that's mentioned in the article you linked here. I never thought of this and frankly, I used to prefer spaces. But because of this I've just changed my local settings.